v2/book/list.html

```
@@ -17,7 +17,7 @@
 </head>
 <body>
     <h1>
-This is the list of 279 RSCG with examples =>
+This is the list of 280 RSCG with examples =>
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)